### PR TITLE
Migrate sensors and input controllers to Svelte

### PR DESCRIPTION
**Pull request by ethanlook**
Migrates sensors and input controllers from Vue to Svelte.

Tested locally, but I'd appreciate suggestions for ways to validate functionality!

### CODE REVIEW

File: web/frontend/src/components/input-controller/index.svelte [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  ];
+
+  $: connected = ((eventsList: inputControllerApi.Event.AsObject[]) => {
+    for (const { event } of eventsList) {
+      if (event !== "Disconnect") {
+        return true;
+      }
+    }
+    return false;
+  })(status.eventsList);
```
Comment (micheal-parks) [inline web/frontend/src/components/input-controller/index.svelte:37]: perhaps could be refactored to:

```suggestion
$: connected = status.eventsList.some(({ event }) => event !== 'Disconnect')

**Review by micheal-parks — APPROVED** [review]
🚀

File: web/frontend/src/components/input-controller/index.svelte (hunk 1 above)
Comment (ethanlook) [reply web/frontend/src/components/input-controller/index.svelte:37]: Works for me! Happy to simplify and improve readability.

Comment (ethanlook) [reply web/frontend/src/components/input-controller/index.svelte]: Done.

**Pull request merged**